### PR TITLE
Add price field in item

### PR DESCRIPTION
**Pull request by yixiann**
Fixes: https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/225

### CODE REVIEW

File: src/main/java/seedu/foodrem/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -11,12 +11,6 @@ public class CliSyntax {
...
     public static final Prefix PREFIX_ITEM_EXPIRY_DATE = new Prefix("exp/");
+    public static final Prefix PREFIX_ITEM_PRICE = new Prefix("p/");
     public static final Prefix PREFIX_ID = new Prefix("id/");
-
-    /* Definitions for Sort Command options */
-    public static final Prefix PREFIX_SORT_BY_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_SORT_BY_QTY = new Prefix("qty/");
-    public static final Prefix PREFIX_SORT_BY_BOUGHT_DATE = new Prefix("bgt/");
-    public static final Prefix PREFIX_SORT_BY_EXPIRY_DATE = new Prefix("exp/");
-    public static final Prefix PREFIX_SORT_BY_UNIT = new Prefix("unit/");
```
Comment (yixiann) [inline src/main/java/seedu/foodrem/logic/parser/CliSyntax.java:21]: I believe this is not needed. When we change a prefix in future, we would like to sort items by the new prefix as well.

File: src/main/java/seedu/foodrem/model/item/itemvalidators/ItemBoughtDateValidator.java [hunk 2]
```diff
@@ -6,65 +6,42 @@
...
+import seedu.foodrem.commons.util.ValidationUtil;
 
 /**
  * Validation class for item dates.
  */
 public class ItemBoughtDateValidator implements Validator {
 
     // Validation for parsing
-    public static final String MESSAGE_FOR_UNABLE_TO_PARSE_BOUGHT_DATE =
-            String.format("The item bought date must follow the format %s.", BOUGHT_DATE_PATTERN_REGEX);
```
Comment (yixiann) [inline src/main/java/seedu/foodrem/model/item/itemvalidators/ItemBoughtDateValidator.java:19]: The regex is dd-MM-uuuu. The user does not know what is uuuu. Thus the change

**Comment by yixiann** [comment]
While performing test on item fields, I modified some quantity validation to prevent unnecessary usage of "public static final". I created a class in the testUtil called MessageToUser that holds all public static final fields.

File: src/main/java/seedu/foodrem/commons/util/ValidationUtil.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+public class ValidationUtil {
+
+    /**
+     * Returns true if a string can be parsed into a double, false otherwise.
+     *
+     * @param string a string to be parsed.
+     */
+    public static boolean isParsableDouble(String string) {
+        try {
+            Double.parseDouble(string);
```
Comment (RichDom2185) [inline src/main/java/seedu/foodrem/commons/util/ValidationUtil.java:20]: Let us make the parameter name more clear.

```suggestion
     * @param doubleString a string to be parsed.
     */
    public static boolean isParsableDouble(String doubleString) {
        try {
            Double.parseDouble(doubleString);
```

**Review by RichDom2185 — DISMISSED** [review]
LGTM, thanks!

File: src/main/java/seedu/foodrem/commons/util/ValidationUtil.java (hunk 3 above)
Comment (yixiann) [reply src/main/java/seedu/foodrem/commons/util/ValidationUtil.java]: Thanks for the suggestion!

**Pull request merged**